src/styles/blocks/blocks.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__info {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color;
+    margin-bottom: 4px;
+    margin-top: 0;
+
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+    margin-bottom: 16px;
+
+  }
+
+  &__rating {
+    @include space-between-content;
+    margin-bottom: 24px;
+
+    &-count {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: $main-color;
+    }
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    @include space-between-content;
+
+    &-text {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: $secondary-color;
+    }
+
+    &-number {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $main-color;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    padding: 12px;
+    background-color: $blue-color;
```

Comment:
Use a variable for the background-color, so it can be easily changed. For example: `background-color: $button-bg-color;`.